### PR TITLE
Issue#28 Added component ToggleButton

### PR DESCRIPTION
**Pull request by OlkaA**
Added new component ToggleButton with responsive design

### CODE REVIEW

**Review by laila1222 — APPROVED** [review]
Looks good. 👍

File: src/client/stories/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { Button } from "@storybook/react/demo";
+import { ToggleButton } from "../components/ToggleButton/ToggleButton.story";
 
 storiesOf("Button", module)
```
Comment (bytewiz) [inline src/client/stories/index.js]: Can we remove this please it is no longer needed ✌️

File: src/client/stories/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { Button } from "@storybook/react/demo";
```
Comment (bytewiz) [inline src/client/stories/index.js:3]: Can we remove this please it is no longer needed ✌️

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good minor comment 👍

File: src/client/stories/index.js (hunk 1 above)
Comment (OlkaA) [reply src/client/stories/index.js]: Done!

File: src/client/components/ToggleButton/ToggleButton.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './ToggleButton.css';
+
+const ToggleButton = ({ }) => {
```
Comment (bytewiz) [inline src/client/components/ToggleButton/ToggleButton.jsx]: @OlkaA As mentioned if you can also expose the `value` for the "switch/toggle button" so we can get the value from outside of the component that will be perfect 👌 so we have a way to figure out which of the options are the selected one 👍

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/components/ToggleButton/ToggleButton.jsx (hunk 3 above)
Comment (OlkaA) [reply src/client/components/ToggleButton/ToggleButton.jsx]: Exposed the value fot ToggleButton.
Added test.
Resolved merged conflicts.

**Review by zkwsk — APPROVED** [review]

**Review by bytewiz — APPROVED** [review]
💥 Looks good!

**Pull request merged**